validator/specs/lib/validateB3dmSpec.js

```diff
@@ -1,9 +1,27 @@
 'use strict';
+var Cesium = require('cesium');
 var validateB3dm = require('../../lib/validateB3dm');
 
+var loadJson = Cesium.loadJson;
+
 describe('validateB3dm', function() {
+    var batchSchema;
+    beforeAll(function(done) {
+        console.log("beforeAll");
 
+        loadJson('https://raw.githubusercontent.com/AnalyticalGraphicsInc/3d-tiles/master/schema/batchTable.schema.json')
+            .then(function(schema) {
```

Comment:
We discussed this offline, that using http.request (or alternatives) is fine to replace loadJson.